src/ott/geometry/grid.py

```diff
@@ -298,17 +299,6 @@ def transport_from_potentials(
         ' cloud geometry instead'
     )
 
-  def apply_transport_from_potentials(
```

Comment:
Shouldn't this be removed from `Graph` as well?

Reply:
removed it too